src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataServiceSpec.scala

```diff
@@ -1,6 +1,7 @@
 package uk.gov.nationalarchives.tdr.api.service
```

Comment:
Could you add or update a RouteSpec test, just to make sure things work end-to-end?

Reply:
I've added one to check for the metadata in `ConsignmentRouteSpec` I didn't want to add it to any of the existing ones as they're already pretty  busy.